pages/Nether-Ice.md

```diff
@@ -0,0 +1,7 @@
+Nether Ice is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used to create [Enriched Nether Ice](https://github.com/Slimefun/Slimefun4/wiki/Enriched Nether Ice), a powerful reactor coolant.
+
+## Obtaining
+Nether Ice must be obtained through a [GEO Miner](https://github.com/Slimefun/Slimefun4/wiki/GEO-Miner).
```

Comment:
Could you add that Nether Ice is only obtainable in the Nether? Or should it not be added since the name implies it is from the Nether?

Reply:
Ah, my bad, I have no idea where to find these comments. I can see that you made them but their location remains a mystery.